### DISCUSSION TITLE
Scanner with default OpenVas won't run, cant access /opt/gvm/var/run/ospd.sock

### DISCUSSION THREAD

**Discussion by nadavbuc** [Secure-Compliance-Solutions-LLC/GVM-Docker#139] [General]
**Describe the bug**
When trying to launch a scanner - it returns an error, in the logs you can see that it didn't had access to the ospd.sock

**To Reproduce**
Try running a scan

**Expected behavior**
Report is generated

**Actual Behaviour**
Report returns an error

**Additional context**
Fixeable by changing the ospd run line - and adding -m flag:
`ospd-openvas --log-file /usr/local/var/log/gvm/ospd-openvas.log --unix-socket /var/run/ospd/ospd.sock --log-level INFO -m 0o777`
And also
```shell
mkdir -p /opt/gvm/var/run/
ln -s /var/run/ospd/ospd.sock /opt/gvm/var/run/ospd.sock
```
I added these to start.sh file locally and everything worked like a charm

**Comment by pixelsquared** [comment]
Are the directory and file link required for it to work for you?

**Comment by nadavbuc** [comment]
Yes, or i can modify the scanner to use the original sock path, but its manual after the container is up

**Comment by pixelsquared** [comment]
I just don't understand why your setup is looking for the socket at /opt/gvm/var/run/ospd.sock

**Comment by austinsonger** [comment]
I'm converting to a discussion. This doesn't seem to be an issue for the majority of users.

**Comment by nadavbuc** [comment]
Sorry for very late response
Im trying it now from scratch on a new server
I'll let you know if it reoccours

**Reply by austinsonger** [reply]
Any update?

**Reply by Masaya-A** [reply]
I think this should be fixed with #143

**Reply by austinsonger** [reply]
If you can please upvote or downvote on [feature requests](https://github.com/Secure-Compliance-Solutions-LLC/GVM-Docker/discussions/categories/feature-request), or you can even submit your own feature request.

**Comment by nadavbuc** [comment]
It works for me
🔥🔥🔥

**Reply by austinsonger** [reply]
If you can please upvote or downvote on [feature requests](https://github.com/Secure-Compliance-Solutions-LLC/GVM-Docker/discussions/categories/feature-request), or you can even submit your own feature request.
